README.rst

```diff
@@ -442,6 +442,74 @@ In order to switch to this test runner you have set the following in your `setti
 
     TEST_RUNNER = 'openwisp_utils.tests.TimeLoggingTestRunner'
 
+``openwisp_utils.tests.redirect_stdout``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This decorator can be used to redirect standard output produced by tests to somewhere else.
+
+Usage example as a decorator:
+
+.. code-block:: python
+
+    import io
+    from openwisp_utils.tests import redirect_stdout
+
+    @redirect_stdout(io.StringIO())
```

Comment:
I understand `io.StringIO()` is not necessary (I see you call the decorator in the tests without doing this), therefore I suggest showing examples in a way that users won't be lead to believe they have to do this.

I would then mention that StringIO is used behind the scenes to capture output and if users need they can pass their own StringIO instance.

Applies to other occurrences.